syntax = "proto2";

package ecclesia;

message QueryFlag {
  optional bool devpath_enabled = 1 [default = false];
  optional string hostname = 2 [default = "localhost"];
  optional int32 port = 3 [default = 8080];
  optional string transport = 4 [default = "http"];
  optional string cert_chain = 5;
  optional string private_key = 6;
  optional string root_cert = 7;
  optional string input_dir = 8;
  optional string query_rule_location = 9;
  repeated string query_id = 10;
}
